gpii/node_modules/testing/src/CloudBasedOAuth2.js

```diff
@@ -64,10 +64,17 @@ gpii.test.stringifyFormBody = function (formBody) {
     return querystring.stringify(encoded);
 };
 
-// On startup of the datastore, overwrite its model so that the stored GPII user token
+// On startup of the datastore, overwrite its content so that the stored GPII user token
 // and authDecisions agree with the data in the current test fixture
-gpii.test.cloudBased.oauth2.populateDataStore = function (dataStore, testCaseHolder) {
+gpii.test.cloudBased.oauth2.populateDataStore = function (testCaseHolder, doneEvent) {
+
+    // TODO: What's the best way get access to and modify the data store?
+    var db = testCaseHolder.pouchHarness.pouch.pouch.databaseInstances.gpiiOauth;
```

Comment:
You can leave a note here to explain that this should be supplied via an IoC-resolved argument, which we will be able to do after the FLUID-4892 rewrite of Infusion. You can see some attempts to solve similar problems in GPII's own testing framework here:
https://github.com/GPII/universal/blob/master/gpii/node_modules/testing/src/Testing.js#L174
and here:
https://github.com/amb26/universal/blob/GPII-580/gpii/node_modules/testing/src/Testing.js#L281

you could do something similar here but you might decide that it is not worth it.